reactDashboard/src/components/DataCharts/DataChartsUtils.test.js

```diff
@@ -104,3 +203,104 @@ describe("calculateDateRange", () => {
     expect(result).toStrictEqual(EMPTY_DATE_RANGE);
   });
 });
+
+describe("getChartYMax", () => {
+  const TEST_DATE_RANGE = {
+    startDate: Date.parse("2020-01-01"),
+    endDate: Date.parse("2020-01-08"),
+  };
+  const EMPTY_DATE_RANGE = {
+    startDate: Date.parse("1980-01-01"),
+    endDate: Date.parse("1980-01-08"),
+  };
+
+  const TEST_EMPTY_DATASET = [
```

Comment:
and here:
  const TEST_EMPTY_DATASET = [{ data: [] }];
  const TEST_SINGLE_DATASET = [{ data: TEST_MAX_DATA_1 }];
  const TEST_DOUBLE_DATASET = [
    { data: TEST_MAX_DATA_1 },
    { data: TEST_MAX_DATA_2 },
  ];